Makefile

```diff
@@ -171,6 +171,12 @@ deploy-mutation: patch-image
 	kustomize build config/overlays/mutation_webhook | kubectl apply -f -
 	kustomize build --load_restrictor LoadRestrictionsNone config/overlays/mutation | kubectl apply -f -
 
+undeploy-mutation:
```

Comment:
```suggestion
uninstall-mutation:
```

Reply:
Done

Reply:
For some reason, I do not see the changes in the latest commit.